package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/feedhenry/grunt-fh-build/issues"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "6.9"
```

Comment:
This seems very high. Is it plugins that are forcing this jump? I'm just wary that fh-mbaas-api uses this module as a devDependency, and I think we'd need to be able to support nodejs@4 for a while with that yet, right? I guess we could ensure that we always run the build job with 6.9+, but then we should probably still be running the tests with 4 also, to ensure we don't break that by using something new that's only in later versions. WDYT?